app/routes/planet_routes.py

```diff
@@ -13,8 +43,33 @@ def get_all_planets():
 @planets_bp.get("/<planet_id>")
 def get_one_planet(planet_id):
     planet = validate_planet(planet_id)
-    return planet.to_dict(),200
-    
+    return planet.to_dict()
+
+@planets_bp.get("/count")
+def get_count():
```

Comment:
👍 Nice custom end point to retrieve the number of planets. We need to be a little careful when our endpoint matches the same pattern as a placeholder route to make sure this is detected properly. In some frameworks (and even in some versions of flask), trying to map `/planets/count` could end up matching `/planets/<planet_id>`, with `count` becoming the `planet_id` value. Some times this is order dependent, and sometimes the framework will treat a literal value like this as "higher priority." So be sure to understand why this _could_ cause issues, and then double check in your framework what the actual behavior is (espcially nice if you can find documentation about the scenario).